### DISCUSSION TITLE
ArgoCD behind AWS ALB SSO

### DISCUSSION THREAD

**Discussion by omegion** [argoproj/argo-cd#13036] [Q&A]
Hi 👋🏼 

We are running our ArgoCD behind [AWS ALB SSO integrated with Okta](https://medium.com/@jainendramandavi/okta-oidc-integration-with-aws-application-load-balancer-alb-5f6e03bbb66b).

In this setup, we cannot use ArgoCD CLI, it fails with:
```
FATA[0003] rpc error: code = Unknown desc = unexpected HTTP status code received from server: 464 (); malformed header: missing HTTP content-type 
```

Is there any way to make CLI work with ALB SSO?

Bests,
Hakan

**Comment by ArjunDandagi** [comment]
you can't query any application via cli (ofc its possible to write a custom client that does the authentication and retrive the token) behind the AWS ALB SSO. 

you should instead. integrate sso in Argocd itself . and not use authentication in the ingress configuration
